dev-lang/ispc/ispc-1.14.0.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
Does 3.9 work?

Reply:
There shouldn't be any issue that I know of.

So I guess I'll just add it?

Reply:
Does it work with it and do tests pass, ideally?

Reply:
It seems to be working and the tests pass.